File: 백준/Bronze/1264. 모음의 개수/모음의 개수.py
# 시험기간
import sys

while True:
    s = sys.stdin.readline().rstrip()
    if s == '#':
        break
    else:
         cnt = 0
         for i in s:
             if i == 'A' or i == 'E' or i == 'I' or i == 'O' or i == 'U' or i == 'a' or i == 'e' or i == 'i' or i == 'o' or i == 'u':
                 cnt += 1

         print(cnt)

File: 백준/Bronze/10987. 모음의 개수/모음의 개수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    static BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
    static StringTokenizer token;
    public static void main(String[] args) throws IOException {
        String s = input.readLine();
        char[] chars = s.toCharArray();
        int cnt = 0;
        for (char c : chars) {
            if (c=='a'||c=='e'||c=='i'||c=='o'||c=='u') {
                cnt++;
            }
        }
        System.out.println(cnt);
    }
}